src/utils/types.tsx

```diff
@@ -10,10 +10,3 @@ export type IconProps = {
     className?: string;
     style?: React.CSSProperties;
 };
-
-export type ContainerType =
-    | 'desktop-small-column'
-    | 'desktop-medium-column'
-    | 'desktop-wide-column'
-    | 'tablet-column'
-    | 'mobile-column';
```

Comment:
As confirmed with design (@MontseTelefonica ) we should keep support for container types in JS code (not only CSS), because next week we'll require the carousel's itemsPerPage prop to behave differently (to have different values) depending on the containers, including desktop small, medium and wide columns, mobile column and tablet column.

Reply:
ok, added a context